api/swagger.yaml

```diff
@@ -1837,6 +1844,13 @@ definitions:
         description: "The ID of the swarm."
         type: "string"
       Version:
+        description: |
+          The version number of the swarm object being. This is required to avoid conflicting writes.
```

Comment:
@thaJeztah Oh that was from the old PR. The updated PR has changed. It is probably the web cache I guess.